<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>QEMU-GUI</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::MinimumExpanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" rowspan="3">
     <widget class="QListView" name="listView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="creat_button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-radius: 10px;
    background-color: rgb(200, 200, 200);
}

QPushButton:pressed {
	background-color:rgb(180, 180, 180);
}

QPushButton:hover {
    border: 3px solid;
}</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../img/create.png</normaloff>../img/create.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="open_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-radius: 10px;
    background-color: rgb(200, 200, 200);
}

QPushButton:pressed {
	background-color:rgb(180, 180, 180);
}

QPushButton:hover {
    border: 3px solid;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../img/open.png</normaloff>../img/open.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="open_menu"/>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="setting_menu">
    <property name="geometry">
     <rect>
      <x>602</x>
      <y>323</y>
      <width>148</width>
      <height>72</height>
     </rect>
    </property>
    <property name="title">
     <string>工具(&amp;S)</string>
    </property>
    <addaction name="create_menu"/>
   </widget>
   <widget class="QMenu" name="help_menu">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="about_menu"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="setting_menu"/>
   <addaction name="help_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action114">
   <property name="text">
    <string>114</string>
   </property>
  </action>
  <action name="create_menu">
   <property name="text">
    <string>新建虚拟硬盘(&amp;C)</string>
   </property>
  </action>
  <action name="open_menu">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="about_menu">
   <property name="icon">
    <iconset>
     <normaloff>../img/logo128.png</normaloff>../img/logo128.png</iconset>
   </property>
   <property name="text">
    <string>关于 QEMU-GUI(&amp;A)</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>新建(&amp;C)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
